### ISSUE TITLE
[Bug]: No error msg if no wallet is connected checkout V2

### ISSUE THREAD

**Issue by Bangalisch** [btcpayserver/btcpayserver#4519]
### What is your BTCPay version?

1.7.3

### How did you deploy BTCPay Server?

lunanode

### What happened?

Deployed new instance, new store, made a POS without yet connecting (creating) a wallet for the store. 
Whenever you now generate invoice (say through a connected shopify) it doesn't pop up the error msg of a missing wallet. 

Low chances of this happening, however, old checkout did give error msg. 

### How did you encounter this bug?

Create store. 
Create POS
Create connection to shopify
DONT connect/create a wallet yet. 

Create invoice and no QR shows nor does it error on not having a wallet connected. 

### Additional information

![SATSBACKBUG](https://user-images.githubusercontent.com/102448109/211577005-abd4b56e-ba7a-4f5a-8cf5-80524f51c2a2.jpg)

### Are you sure this is a bug report?

- [X] I confirm this is a bug report

**Comment by pavlenex** [comment]
How were you able to create an app before creating a wallet, and then how were you able to generate an invoice without even having a wallet, my understanding is that these actions shouldn't be possible, or am I misunderstanding your bug report @Bangalisch ?

**Comment by Bangalisch** [comment]
These actions are fully possible.

**Comment by pavlenex** [comment]
Yeah, that's what I was thinking, @Bangalisch can you either provide more info or exact steps, because it's not making much sense.

**Comment by bolatovumar** [comment]
I couldn't repro. I created a new store with no wallet and set up a Shopify integration and I get an error when I try to check out. This happens both with the new and the old versions of the checkout page. As soon as I create a wallet the error goes away.

![Capture](https://user-images.githubusercontent.com/1934678/219551915-52f4d90d-acfd-4a42-b602-c1dd20bfb730.PNG)

I followed this guide https://docs.btcpayserver.org/Shopify/
